lib/PuppeteerSharp.Tests/PageTests/SetRequestInterceptionTests.cs

```diff
@@ -458,7 +458,7 @@ public async Task ShouldNotThrowInvalidInterceptionIdIfTheRequestWasCancelled()
                 requestIntercepted.SetResult(true);
             };
 
-            var _ = Page.QuerySelectorAsync("iframe").EvaluateFunctionAsync<object>("(frame, url) => frame.src = url", TestConstants.ServerUrl);
+            var _ = (await Page.QuerySelectorAsync("iframe")).EvaluateFunctionAsync<object>("(frame, url) => frame.src = url", TestConstants.ServerUrl);
```

Comment:
The original test is not awaiting this function https://github.com/GoogleChrome/puppeteer/blob/7f9e276733a7dea8ccc414147a637862884403c3/test/network.spec.js#L554